cola/models/stash.py

```diff
@@ -89,10 +90,12 @@ def do(self):
         status, out, err = git.stash('drop', self.stash_ref)
         if status == 0:
             Interaction.log_status(status, out, err)
+            return re.search('\\((.*)\\)', out).group(1)
```

Comment:
Please add a comment above the regex showing an example of the string it's trying to parse.  That way we can spot-check it in the future.

It's not reliable to rely on the output of `git stash drop` like this, so we shouldn't blindly assume that the regex matches.  It's better to assign to a variable and handle `None` since we can't predict what the output will look like in the future.  The output of `git stash` is "porcelain" (it's not "plumbing" in git jargon) so the output is not guaranteed to be stable between versions.  Thus, we have to be robust.

For the regex itself, please use `r'aw strings'` so that we don't have to escape the backslashes.  Also, named groups would make this a bit easier to grok.  Maybe something like this?

```
result = ''
match = re.search(r'\((?P<name>.*)\)', out)
if match:
    result = match.group('name')
```

Reply:
Looks good to me! I will try that.